contracts/UBI.sol

```diff
@@ -146,6 +207,16 @@ contract UBI is Initializable {
     governor = _governor;
   }
 
+  /** @dev Changes `accruedPerSecond`
+  *  @param _accruedPerSecond.
+  * IF changed totalSupply will be +- of real total supply
+  */
+  function changeAccruedRate(uint256 _accruedPerSecond) external onlyByGovernor {
+    supplyMarker += ((activeVerifiedHumans * accruedPerSecond) * (block.timestamp - lastSupplyUpdate));
+    lastSupplyUpdate = block.timestamp;
+    accruedPerSecond = _accruedPerSecond;
+  }
+
```

Comment:
Changing the accrued rate would require additional changes when making calculations with `accruedSince`. Perhaps best to remove this method and make it a separate PR later.

Reply:
It's actually here to show/`test` why we can't have 100% outgoing in stream, and side effects calculating real `totalSupply` if/when it's changed.

Reply:
Changing `accruedPerSecond` would require recording the old rate and the block when the rate is changed. Then everywhere `accruedSince` is called would need to check the block and apply the different rates accordingly. It is complexity that is orthogonal to where the UBI is accumulating (and s/b out of scope for this PR). I don't see any objective reason to limit the percentage a person chooses to stream, it should be their sovereign choice just like they can transfer any amount freely. You're right, this could always be addressed by vote in the DAO later if a limit makes it in.